pkg/gardenlet/controller/shoot/shoot_control_delete.go

```diff
@@ -351,6 +351,16 @@ func (c *Controller) runDeleteShootFlow(o *operation.Operation, errorContext *er
 			Fn:           botanist.WaitUntilExtensionResourcesDeleted,
 			Dependencies: flow.NewTaskIDs(deleteExtensionResources),
 		})
+		deleteContainerRuntimeResources = g.Add(flow.Task{
+			Name:         "Delete stale container runtime resources",
+			Fn:           botanist.DeleteContainerRuntimeResources,
+			Dependencies: flow.NewTaskIDs(initializeShootClients),
```

Comment:
```suggestion
			Dependencies: flow.NewTaskIDs(initializeShootClients, cleanKubernetesResources, cleanShootNamespaces),
```

I guess the container runtimes should only be deleted after the Kubernetes cluster has been cleaned, right?

Reply:
Yes